config/redirects.yml

```diff
@@ -337,7 +337,7 @@ redirects:
   "/urgent-call-for-qualified-teachers": "/returning-to-teaching"
 
   # Blog pages
-  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/blog/5-reasons-to-attend-a-get-into-teaching-event"
+  "/blog/5-reasons-to-attend-a-get-into-teaching-event" : "/events/about-get-into-teaching-events"
   "/blog/teachers-pension-scheme": "/is-teaching-right-for-me/what-pension-does-a-teacher-get"
```

Comment:
We should probably also add a redirect for the blog tag that has been deleted, for completeness:

FROM /blog/tag/get-into-teaching-events
TO /events/about-get-into-teaching-events